src/ui/App.swift

```diff
@@ -69,91 +80,103 @@ class App: NSApplication, NSApplicationDelegate {
         // pre-load tabs so we can interact with them before the user opens the preferences window
         tabs.forEach { (tab: NSViewController) in tab.loadView() }
         preferencesWindowController = PreferencesWindowController(preferencePanes: tabs as! [PreferencePane])
+        
+        if let window = preferencesWindowController.window {
```

Comment:
You can replace the `if let` with `preferencesWindowController!.window` here, as we know it is available by that point.